stetho-sample/src/debug/java/com/facebook/stetho/sample/SampleDebugApplication.java

```diff
@@ -28,10 +36,33 @@ public void onCreate() {
 
     long startTime = SystemClock.elapsedRealtime();
     final Context context = this;
+
+    // Setup a javascript runtime
```

Comment:
Only 2 lines are mandatory:

    JavaScriptRuntimeBuilder jsRuntimeBuilder = new JavaScriptRuntimeBuilder(context);
    Stetho.newInitializerBuilder(context).enableWebKitInspector(jsRuntimeBuilder.javaScriptInspectorModulesProvider());

The other lines where adding to show everything that can be done:
 - import a class
 - import a package
 - add a variable
 - add a custom function

What I should do instead is to add these code snippets to the documentation. Let me rework the patch with documentation & screenshots. I'll also comment on the dex increase, apk size, proguard settings.